### DISCUSSION TITLE
Get the count of the rows filtered

### DISCUSSION THREAD

**Discussion by EswarNandam** [TanStack/table#2127] [General]
How do I get the count of the filtered rows and use that for displaying?

**Comment by ayushmalviya27** [comment]
I think `rows.length` works for this.

**Reply by Unviray** [reply]
Where can i access `rows` ?

**Reply by felipesnts** [reply]
If you're using v8 it looks like you can grab that value using `table.getFilteredRowModel().rows.length`

**Reply by chandu535** [reply]
That's oK 
by for me the row size is printing 17 and the rows are only 10 on the output screen.
it doesn't printing data more thatn 10 on any cases

**Reply by chandu535** [reply]
@felipesnts

**Reply by MinuraPunchihewa** [reply]
@felipesnts This really helped me! Thanks. I used `table.getSelectedRowModel().rows.length` to get the number of rows that were selected.

**Reply by ThibaultJanBeyer** [reply]
This seems to work also when filtering: `table.getCoreRowModel().rows.length` using the core model.

**Comment by chandu535** [comment]
@felipesnts 
 That's oK by for me the row size is printing 17 and the rows are only 10 on the output screen. it doesn't printing data more thatn 10 on any cases

**Comment by chandu535** [comment]
@felipesnts 
 That's oK by for me the row size is printing 17 and the rows are only 10 on the output screen. it doesn't printing data more thatn 10 on any cases

**Reply by felipesnts** [reply]
That script gives the total number of filtered items and not the number of items on the current page.
